app/javascript/courses/students/components/CoursesStudents__LevelDistribution.re

```diff
@@ -0,0 +1,125 @@
+[%bs.raw {|require("./CoursesStudents__LevelDistribution.css")|}];
+
+open CoursesStudents__Types;
+let str = React.string;
+
+let stylingForLevelPills = percentageStudents => {
+  let emptyStyle = ReactDOMRe.Style.make();
+  let styleWithWidth =
+    ReactDOMRe.Style.make(
+      ~width={
+        (percentageStudents |> Js.Float.toString) ++ "%";
+      },
+      (),
+    );
+  if (0.0 <= percentageStudents && percentageStudents <= 4.0) {
+    ("w-8 flex-shrink-0", emptyStyle, "bg-green-200 text-green-800");
+  } else if (4.0 < percentageStudents && percentageStudents <= 20.0) {
+    ("", styleWithWidth, "bg-green-300 text-green-800");
+  } else if (20.0 < percentageStudents && percentageStudents <= 40.0) {
+    ("", styleWithWidth, "bg-green-400 text-green-900");
+  } else if (40.0 < percentageStudents && percentageStudents <= 60.0) {
+    ("", styleWithWidth, "bg-green-500 text-white");
+  } else if (60.0 < percentageStudents && percentageStudents <= 80.0) {
```

Comment:
0 to 4 and 4 to 20? Should this be 0 to 5 and 5 to 20? 🤔 

Also, these checks feel just a _bit off_.

```suggestion
  if (0.0 <= percentageStudents && percentageStudents < 5.0) {
    ("w-8 flex-shrink-0", emptyStyle, "bg-green-200 text-green-800");
  } else if (5.0 <= percentageStudents && percentageStudents < 20.0) {
    ("", styleWithWidth, "bg-green-300 text-green-800");
  } else if (20.0 <= percentageStudents && percentageStudents < 40.0) {
    ("", styleWithWidth, "bg-green-400 text-green-900");
  } else if (40.0 <= percentageStudents && percentageStudents < 60.0) {
    ("", styleWithWidth, "bg-green-500 text-white");
  } else if (60.0 <= percentageStudents && percentageStudents < 80.0) {
```